ManikShrivastav.md

```diff
@@ -0,0 +1,45 @@
+---
+Name: “ManikShrivastav”
+Project: “Gesture Robot”
+---
+
+# Project Name
+Gesture Robot
+
+## Summary
+Gesture Robot is an electronic car controlled by the movement of circuit. It can be mostly used in real 
+life robot fights and car fights. It is a car that can be controlled according to the user’s movement of the 
+hands. The transmitter part will be attached to the hand of the user and its receiver will act as the 
+motherboard of the car. As the transmitter moves, it will send the signals to the receiver in the car and it 
+will turn and run the car according to the instructions.
+
+## Plan
+First of all, the chassis of car is going to be made. It will contain four motors and rubber wheels attached 
+to it. After that, the motors will be connected with motor shield driver. After that, the Arduino Nano is 
+connected to the RF module and it acts as a transmitter with the power supply of li-ion battery. The
+other Arduino Nano is connected with RF Module and ADLX335 Module and it acts as a receiver with 
+power supply from 9V Battery. The code would be uploaded to both the Arduinos to make them 
+transmit and receive signals and then the circuit works.
+
+## Budget
+
+| SN   | Product                               | Supplier/Link                         | Cost   |
+| ---- | ------------------------------------- |-------------------------------------- | ------ |
+|  1.  | Arduino Nano                          |  https://amzn.to/3GiAeTd              | 20.46  |
+|  2.  | NRF24L01 RF Module                    |  https://amzn.to/3BXDfFC              | 7.86   |      
+|  3.  | L298N Motor Driver                    |  https://amzn.to/3WqvbFK              | 4.12   |
+|  4.  | ADXL335 Module                        |  https://amzn.to/3HZsmaa              | 8.23   |
+|  5.  | TT Gear Motor                         |  https://amzn.to/3I2uMF8              | 3.15   |
+|  6.  | Rubber Wheels                         |  https://amzn.to/3hO8MmD              | 2.30   |
+|  7.  | 2.5mm Male Female Strip PCB Connector |  https://amzn.to/3Vlfj6c              | 2.06   |
+|  8.  | 18650 li-ion Battery                  |  https://amzn.to/3C1zwqF              | 10.04  |
+|  9.  | 18650 battery Holder(2 Slot)          |  https://amzn.to/3WIi3eR              | 1.80   |
+|  10. | Screw terminal (2 pin)                |  https://amzn.to/3vdaHnW              | 2.06   |
+|  11. | Beard Board                           |  https://amzn.to/3vgWEhi              | 2.30   |
+|  12. | Acrylic sheet (25cm*15cm)             |  https://amzn.to/3YKAmlr              | 3.27   |
```

Comment:
What will the acrylic be used for?

Reply:
I am planning to make a transparent outer cover to protect the internal Circuit so it for that purpose.